### DISCUSSION TITLE
Release v3.0.0

### DISCUSSION THREAD

**Discussion by ZigRazor** [ZigRazor/CXXGraph#369] [Releases]
## What's Changed
* Added reverseDirectedGraph() function by @Grufoony in https://github.com/ZigRazor/CXXGraph/pull/320
* Fix bug in overload of addEdge by @sbaldu in https://github.com/ZigRazor/CXXGraph/pull/321
* Bump thomaseizinger/create-pull-request from 1.3.0 to 1.3.1 by @dependabot in https://github.com/ZigRazor/CXXGraph/pull/323
* Create windows-cmake.yml by @ZigRazor in https://github.com/ZigRazor/CXXGraph/pull/327
* Add tests for cycles in directed graphs by @sbaldu in https://github.com/ZigRazor/CXXGraph/pull/325
* Migrate from CMake FetchContent to CMake Package Manager (CPM) by @perhapsmaple in https://github.com/ZigRazor/CXXGraph/pull/332
* Edge equality operator is incorrect (#333) by @nrkramer in https://github.com/ZigRazor/CXXGraph/pull/326
* introduce tarjan's algorithm (#103) by @suncanghuai in https://github.com/ZigRazor/CXXGraph/pull/322
* Reduce warnings from CMake by @nrkramer in https://github.com/ZigRazor/CXXGraph/pull/334
* Added changes to save shortest path nodes for Dijkstra shortest path. Modified tests. by @guru2396 in https://github.com/ZigRazor/CXXGraph/pull/336
* Bump actions/checkout from 3 to 4 by @dependabot in https://github.com/ZigRazor/CXXGraph/pull/343
* close #345 fixed Node<T> move constructor by @AryanGitHub in https://github.com/ZigRazor/CXXGraph/pull/346
* [Fix] [Issue 338] Inconsistent id types by @nrkramer in https://github.com/ZigRazor/CXXGraph/pull/342
* Caching Adjacency Matrix for faster edge lookup by @guru2396 in https://github.com/ZigRazor/CXXGraph/pull/344
* Update README.md by @ZigRazor in https://github.com/ZigRazor/CXXGraph/pull/349
* Feature node storage for Graph class by @sbaldu in https://github.com/ZigRazor/CXXGraph/pull/347
* Create PaperToTakeIntoConsideration.txt by @ZigRazor in https://github.com/ZigRazor/CXXGraph/pull/337
* fixed make install, close #335 by @AryanGitHub in https://github.com/ZigRazor/CXXGraph/pull/354
* Update README.md by @ZigRazor in https://github.com/ZigRazor/CXXGraph/pull/355
* Feature in/out Edges convenience functions by @sbaldu in https://github.com/ZigRazor/CXXGraph/pull/357
* Use cached adjacency matrix in all Graph algorithms by @sbaldu in https://github.com/ZigRazor/CXXGraph/pull/358
* Fix construction of weighted Edge in addEdge overload by @sbaldu in https://github.com/ZigRazor/CXXGraph/pull/360
* Implement generic addNodes and addEdges by @sbaldu in https://github.com/ZigRazor/CXXGraph/pull/363
* Update README.md by @kamari-a in https://github.com/ZigRazor/CXXGraph/pull/329
* changing output format of Kosaraju algorithm by @guru2396 in https://github.com/ZigRazor/CXXGraph/pull/365
* Implement matrices used in network dynamics by @carlosfrodrigues in https://github.com/ZigRazor/CXXGraph/pull/366
* Preparing stable release 3.0.0 by @ZigRazor in https://github.com/ZigRazor/CXXGraph/pull/368

## New Contributors
* @Grufoony made their first contribution in https://github.com/ZigRazor/CXXGraph/pull/320
* @perhapsmaple made their first contribution in https://github.com/ZigRazor/CXXGraph/pull/332
* @guru2396 made their first contribution in https://github.com/ZigRazor/CXXGraph/pull/336
* @AryanGitHub made their first contribution in https://github.com/ZigRazor/CXXGraph/pull/346
* @kamari-a made their first contribution in https://github.com/ZigRazor/CXXGraph/pull/329
* @carlosfrodrigues made their first contribution in https://github.com/ZigRazor/CXXGraph/pull/366

**Full Changelog**: https://github.com/ZigRazor/CXXGraph/compare/v2.0.0...v3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ZigRazor/CXXGraph/releases/tag/v3.0.0'>Release v3.0.0</a>.</em>

**Comment by YLuc** [comment]
Microsoft vcpkg still lists its version 2.0.0, which is too old. Could you activate their updating to 3.0.0?

**Reply by ZigRazor** [reply]
can you open an issue for this? thank you!
